system/views/renderers/asset/icon.cfm

```diff
@@ -1,6 +1,6 @@
 <cfscript>
 	ext     = ListLast( args.storage_path ?: "", "." );
-	iconUrl = event.buildLink( systemStaticAsset = "/images/asset-type-icons/16px/#LCase( ext )#.png" );
+	iconUrl = event.buildLink( assetId=args.id ?: "", derivative='pickerIcon' );
```

Comment:
This isn't quite right, unfortunately. This will break a lot of other places that can't render an image for assets such as PDFs, etc. 

I think the change that needs to happen here is the code that renders the asset for the picker or, the code that you have here should be made in `/renderers/image/icon.cfm` (note the different folder).